### PR TITLE
SWAP-1306 As a user I want to see when my proposal has been scheduled

### PR DESCRIPTION
**Pull request by peter-asztalos**
## Description

This PR adds support for Users to see their upcoming (or all) beam times.

## How Has This Been Tested

- [x] manual
- [x] e2e test 

## Fixes

https://jira.esss.lu.se/browse/SWAP-1306

## Changes

- support to access scheduler
- core and scheduler github actions updated
- core and scheduler docker compose files updated
- API gateway updated

## Tests included/Docs Updated?

- [ ] I have added tests to cover my changes.
- [ ] All relevant doc has been updated

### CODE REVIEW

File: cypress/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM cypress/included:5.3.0
+FROM cypress/included:6.6.0
```
Comment (martin-trajanovski) [inline cypress/Dockerfile:1]: I know @jekabs-karklins mentioned that there were some issues with this 6.6.0 cypress version on the user-office-frontned but maybe that is not the case here.

File: package.json [hunk 2]
```diff
@@ -90,7 +90,7 @@
     "@types/node": "^12.19.8",
     "@typescript-eslint/eslint-plugin": "~2.34.0",
     "@typescript-eslint/parser": "~2.34.0",
-    "cypress": "5.3.0",
+    "cypress": "6.6.0",
```
Comment (martin-trajanovski) [inline package.json:93]: Maybe this should be another task but just to mention it here that I think it is a good idea to upgrade prettier and eslint to the same version as user-office-frontend on both scheduler-frontend and scheduler-backend just to align the versions and rules everywhere.

**Review by martin-trajanovski — APPROVED** [review]

File: cypress/Dockerfile (hunk 1 above)
Comment (peter-asztalos) [reply cypress/Dockerfile:1]: Yes, I fixed it, the cypress package.json had some missing dependencies

File: package.json (hunk 2 above)
Comment (peter-asztalos) [reply package.json:93]: Yes, would be great but the last time I checked the react-calendar used some deprecated React features so upgrading the React version maybe not possible yet.

**Review by Scott-James-Hurley — APPROVED** [review]

**Review by jekabs-karklins — APPROVED** [review]

**Pull request merged**